asv_benchmarks/benchmarks/ensemble.py

```diff
@@ -75,6 +79,31 @@ def make_estimator(self, params):
                                                subsample=0.5,
                                                random_state=0)
 
+
+class HistGradientBoostingClassifierBenchmark(Predictor, Estimator, Benchmark):
+    """
+    Benchmarks for HistGradientBoostingClassifier.
+    """
+
+    param_names = []
+    params = ()
+
+    def setup_cache(self):
```

Comment:
if all you call is super, this can be omitted :)

Reply:
Currently it's necessary when you have more than one benchmark class in a file that inherit from a base class. It's a known issue of asv, see https://github.com/airspeed-velocity/asv/issues/880

Reply:
@jeremiedbb thank you for informing me, that's a very interesting issue to have! I wonder if some `__init_subclass__` or metaclass magic could fix it